packages/strapi-plugin-content-manager/admin/src/containers/EditPage/index.js

```diff
@@ -410,26 +546,44 @@ export class EditPage extends React.Component {
     );
   };
 
-  toggle = () => this.setState(prevState => ({ showWarning: !prevState.showWarning }));
+  toggle = () =>
+    this.setState(prevState => ({ showWarning: !prevState.showWarning }));
 
   toggleDelete = () =>
-    this.setState(prevState => ({ showWarningDelete: !prevState.showWarningDelete }));
+    this.setState(prevState => ({
+      showWarningDelete: !prevState.showWarningDelete,
+    }));
+
+  /**
+   * Render internal and external links
+   * @type {Array} List of all links to display
+   */
+  renderNavLinks = () => {
+    // Add ctm link as list item to external links array to return the entire list
+    let ctmLink = <li key={`${pluginId}.link`}>{this.layoutLink()}</li>;
```

Comment:
Well I think that your `layoutLink` instance can return the link wrapped in a `<li />`

This way this method can be changed to 
```
renderNavLinks = () => [this.layoutLink(), ...this.retrieveLinksContainerComponents()];
```